src/parser/rogue/outlaw/modules/features/RollTheBonesCastTracker.js

```diff
@@ -79,7 +80,7 @@ class RollTheBonesCastTracker extends Analyzer {
     if(!event || !event.classResources){
       return;
     }
-    const cpCost = getResource(event.classResources, RESOURCE_TYPES.COMBO_POINTS.id).cost;
+    const cpCost = getResource(event.classResources, RESOURCE_TYPES.ENERGY);
```

Comment:
if this doesnt cost Combo Points anymore then the variable probably shouldnt be called CpCost

Reply:
This ended up being removed since Roll the Bones duration is a static 30 seconds for 50 energy. I'll be making another run through this module and making sure everything is being calculated properly.